WorldModel/WorldModel/Core/Core.aslx

```diff
@@ -126,6 +126,16 @@
       else {
         JS.setCss ("#status", "border:none")
       }
+      // Pertex: this is new

```

Comment:
You don't need to include comments like this in-line, we can always use Git to see the history of a file.